pkgs/default.nix

```
@@ -26,6 +26,8 @@
   container-hotplug = pkgs.callPackage ./container-hotplug {};
   surfer = pkgs.callPackage ./surfer/default.nix {};
   uf2conv = pkgs.callPackage ./uf2conv.nix {};
+  sv-lang-06 = pkgs.callPackage ./sv-lang.nix {};
```

I think this is more common in nixpkgs:
```suggestion
  sv-lang_6 = pkgs.callPackage ./sv-lang.nix {};
```